.github/workflows/lint-and-test.yaml

```diff
@@ -13,6 +13,18 @@
 
 name: Lint and Test
 
+env:
+  VSOMEIP_RELEASE_URL: https://github.com/COVESA/vsomeip/archive/refs/tags/
+  VSOMEIP_RELEASE_TAG: 3.4.10.tar.gz
+  VSOMEIP_SOURCE_PATH: vsomeip-src
+  VSOMEIP_INSTALL_PATH: vsomeip-install
+
+  # TODO: Need to get a cached version of vsomeip compiled we can refer to here
```

Comment:
We do have a cached version and the variable used for this is `VSOMEIP_INSTALL_PATH` from above. Should delete this TODO.